.travis.yml

```diff
@@ -2,33 +2,44 @@ language: python
 python: 3.7
 sudo: true
 dist: bionic
+
 cache:
-  pip: true
   directories:
-    - $HOME/docker
+    - "$HOME/.cache/pip"
+    - "$HOME/google-cloud-sdk/"
+
 before_cache:
   - chown -R travis:travis $HOME/.cache/pip
-  - docker save -o $HOME/docker/images.tar athenian/api:latest
```

Comment:
An unclean rebase overwrote my caching trick.

Reply:
the problem is we use makefile from https://github.com/athenianco/ci and I did not know how to propagate this caching in convenient way... if you have some good idea how to achieve this I can fix it...

Reply:
@vmarkovtsev Done